data/brands/shop/sports.json

```
@@ -6,6 +6,18 @@
     }
   },
   "items": [
+        {
+      "displayName": "Johnson Fitness & Wellness",
+      "locationSet": {
+        "include": ["au","br","ca","cn","cz","gr","hk","it","jp","kr","my","mx","ro","tw","th","uk","us","vn"]
+      },
+      "tags": {
+        "brand": "Johnson Fitness & Wellness",
+        "brand:wikidata": "Q120785852",
+        "name": "Johnson Fitness & Wellness",
+        "shop": "sports"
```

```suggestion
        "shop": "sports",
        "sport": "fitness"
```
Based on their website, I think it makes sense to add [`sport=fitness`](https://wiki.openstreetmap.org/wiki/Tag%3Asport%3Dfitness)